lib/config/virtual_container_host.go

```diff
@@ -125,6 +125,9 @@ type VirtualContainerHostConfigSpec struct {
 
 	// Allow custom naming convention for containerVMs
 	ContainerNameConvention string
+
+	// default size for root image of image B tree
```

Comment:
Not to be pedantic, but it's a B+tree.  Anyway, the impl is irrelevant.  Just nuke `of...`.